cmake/packaging.cmake

```diff
@@ -0,0 +1,31 @@
+
+#-----------------------------------------------
+# packaging
+#-----------------------------------------------
```

Comment:
Is packaging required at all?

Reply:
No, it is not needed. It was a feature from the original PR. I'll remove it.